.circleci/config.yml

```diff
@@ -56,6 +56,43 @@ jobs:
           paths:
             - snapshot-sender-*.jar
 
+  integration-test:
+    docker:
+      - image: circleci/python:3.7.3
+    steps:
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
+
+      - run:
+          name: "Run the integration test"
+          command: |
+            sudo pip3 install docker-compose
+            sudo apt-get update
+            sudo apt-get install openjdk-8-jdk
+
+            export HBASE_TO_MONGO_EXPORT_VERSION=$(cat ./gradle.properties | cut -f2 -d'=')
+            export DATA_KEY_SERVICE_URL=http://localhost:8090
+            export DATA_KEY_SERVICE_URL_SSL=https://localhost:8091
+            export S3_BUCKET=dummy-bucket
+            export S3_PREFIX_FOLDER=dummy-folder
+            export AWS_DEFAULT_REGION=dummy-region
+            export AWS_ACCESS_KEY_ID=dummy-key
+            export AWS_SECRET_ACCESS_KEY=dummy-secret
```

Comment:
These might not be relevant for snapshot sender. Might need changing according to what is needed.

Reply:
removed HBASE_TO_MONGO_EXPORT_VERSION